src/main/java/com/softteam/vocabuilder/service/dto/CategoryDTO.java

```diff
@@ -0,0 +1,15 @@
+package com.softteam.vocabuilder.service.dto;
+
+import lombok.Data;
+
+import javax.persistence.Column;
+import java.util.UUID;
+@Data
+public class CategoryDTO {
+    private UUID id;
+    private String title;
+
+    private String description;
+
+    private String color;
```

Comment:
a blank line after `id`, and before `@Data` also? 😄 recuerda que hay que tener el codigo lindo jaja